requirements.txt

```diff
@@ -9,3 +9,5 @@ libsass==0.14.5
 python-dotenv==0.8.2
 bgs==0.6.1
 pyyaml==3.13
+celery==4.2.1
+requests-mock==1.5.2
```

Comment:
I realize requests is captured in the 'parent' dockerfile..this might be a good time to capture our need for the requirement, and upgrade!

Reply:
Good idea. Let's go ahead and refactor out hydroshare while we're at it! :-)

Reply:
We can dream!